src/prefabs/selectinput.tsx

```diff
@@ -2,31 +2,340 @@ import * as React from 'react';
 import { BeforeCreateArgs, Icon, prefab } from '@betty-blocks/component-sdk';
 import { SelectInput } from './structures/SelectInput';
 
+export enum DynamicTypeEnum {
+  Variable = 'VARIABLE',
+  Property = 'PROPERTY',
+  MeProperty = 'ME_PROPERTY',
+  Translation = 'TRANSLATION',
+}
+
```

Comment:
We should provide this in a helper prop as a constant. like we do with `BettyInput`.

Reply:
I got rid of `DynamicTypeEnum`, since indeed we use only **`"PROPERTY"`** option from there